packages/squiggle-lang/__tests__/Reducer/Reducer_Dispatch/Reducer_Dispatch_BuiltIn_test.res

```diff
@@ -1,25 +1,25 @@
-module CT = Reducer.CodeTree
-module CTV = Reducer.Extension.CodeTreeValue
-module JsG = Reducer.Js.Gate
+module ExpressionValue = ReducerInterface.ExpressionValue
 
 open Jest
 open Expect
 
 let expectEvalToBe = (expr: string, answer: string) =>
-  Reducer.eval(expr) -> CTV.showResult -> expect -> toBe(answer)
+  Reducer.eval(expr)->ExpressionValue.showResult->expect->toBe(answer)
 
 describe("builtin", () => {
   // All MathJs operators and functions are available for string, number and boolean
   // .e.g + - / * > >= < <= == /= not and or
   // See https://mathjs.org/docs/expressions/syntax.html
   // See https://mathjs.org/docs/reference/functions.html
-  test("-1", () => expectEvalToBe( "-1", "Ok(-1)"))
-  test("1-1", () => expectEvalToBe( "1-1", "Ok(0)"))
-  test("2>1", () => expectEvalToBe( "2>1", "Ok(true)"))
-  test("concat('a','b')", () => expectEvalToBe( "concat('a','b')", "Ok('ab')"))
+  test("-1", () => expectEvalToBe("-1", "Ok(-1)"))
```

Comment:
These tests, in general, are pretty clean and straightforward, kudos

Reply:
:+1:

Reply:
RESOLVED